src/components/modals/ant-management/UpgradeAntModal/UpgradeAntModal.tsx

```diff
@@ -262,16 +231,16 @@ function UpgradeAntModal({
               !accepted
                 ? 'bg-background text-grey'
                 : `animate-pulse ${
-                    progress < 0
+                    !upgrading
                       ? 'bg-primary-thin text-primary'
                       : 'bg-link text-white hover:bg-primary hover:text-black'
                   } `
             } w-full rounded-b-lg p-3 transition-all`}
             onClick={() => upgradeAnts()}
           >
-            {!accepted && progress < 0
+            {!accepted && !upgrading
```

Comment:
```suggestion
            {!accepted
```

Reply:
!upgrading doesn't affect this calculation since you can't ever get into a situation of !accepted and upgrading. Remove to make this less complicated.